src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatedCSSData = {};
+  const splitedSource = sourceString.split(';');
```

Comment:
The split operation should handle cases where there are empty declarations (e.g., consecutive semicolons). Consider using a filter to remove empty strings from the 'splitedSource' array before processing.